File: pset6/mario/more/mario.py
from cs50 import get_int


def main():
    while True:
        h = get_int("Height: ")
        w = h
        if h > 0 and h < 9:
            break

    for i in range(1, h + 1):
        nh = i
        ns = w - nh
        # print spaces without break line
        print(" " * ns, end="")
        # print left hashes
        print("#" * nh, end="")
        # print 2 spaces
        print(" " * 2, end="")
        # print right hashes
        print("#" * nh)


if __name__ == "__main__":
    main()

File: pset1/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int n;
    
    // gets user input
    do
    {
        n = get_int("Height: ");
    } 
    while (n < 0 || n > 23);
    
    for (int i = 0; i < n; i++)
    {
        // prints spaces
        for (int j = i + 1 ; j < n; j++)
        {
            printf(" ");
        }
        // prints #s
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }
        
        // print central spaces
        printf("  ");
        
        // prints #s
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        // prints new line
        printf("\n");
    }
}